master.cfg

```diff
@@ -303,7 +303,7 @@ ubiquity_lxde.addStep(steps.ShellCommand(command=["/usr/bin/sha256sum", compress
 #ubiquity_lxde.addStep(steps.FileUpload(name="Upload Image", workersrc=compressed_image_file, masterdest=compressed_image_master_dest))
 
 c['builders'].append(
-    util.BuilderConfig(name="pi-lxde-testing-image",
+    util.BuilderConfig(name="pi-lxde-xenial-testing-image",
```

Comment:
I'm not sure if we actually still need xenial support, but I guess we can leave it in.